conda/plugins/subcommands/doctor/cli.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+from __future__ import annotations
+
+import argparse
+from pathlib import Path
+
+from ....base.context import context, locate_prefix_by_name
+from ....cli.conda_argparse import add_parser_prefix
+from ....exceptions import CondaError
+from ... import CondaSubcommand, hookimpl
+
+
+def get_parsed_args(argv: list[str]) -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        "conda doctor", description="Display a health report for your environment."
+    )
+    parser.add_argument(
+        "-v",
+        "--verbose",
+        action="store_true",
+        help="generate a detailed environment health report",
+    )
+    add_parser_prefix(parser)
+    args = parser.parse_args(argv)
+
+    return args
+
+
+def validate_prefix(prefix: str) -> str:
+    """
+    Make sure that the prefix is an existing folder
+
+    :raises CondaError: When the prefix does not point to existing folder
+    """
+    prefix_path = Path(prefix)
+
+    if not prefix_path.is_dir():
+        raise CondaError("Provided prefix does not exist.")
+
+    return prefix
+
+
+def get_prefix(args: argparse.Namespace) -> str:
+    """
+    Determine the correct prefix to use provided the CLI arguments and the context object.
+
+    When not specified via CLI options, the default is the currently active prefix
+    """
+    if args.name:
+        return locate_prefix_by_name(args.name)
+
+    if args.prefix:
+        return validate_prefix(args.prefix)
+
+    # context.active_prefix can return None; always return str
```

Comment:
Why are we always returning a string? An empty string is not a valid conda env so it would make sense to notice this and to error out with a message to the user.